src/components/fee-summary/package.json

```diff
@@ -7,6 +7,7 @@
   "dependencies": {
     "@bcrs-shared-components/enums": "^1.0.42",
     "@bcrs-shared-components/interfaces": "^1.0.66",
+    "@bcrs-shared-components/services": "^1.0.0",
```

Comment:
Set this to 0.0.0 also. "Lerna hoist" will take care of updating this (I think).